dissect/target/plugins/os/windows/locale.py

```diff
@@ -53,4 +54,6 @@ def language(self):
     def timezone(self):
         """Get the configured timezone of the system."""
```

Comment:
```suggestion
        """Get the configured timezone of the system in IANA TZ standard format."""
```
Additional suggestion on top of that patch in the comments.